plugins/woocommerce-admin/client/products/tour/block-editor/use-block-editor-tour-options.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+
+export const BLOCK_EDITOR_TOUR_SHOWN_OPTION =
+	'woocommerce_block_product_tour_shown';
+
+export const useBlockEditorTourOptions = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const [ isGuideOpen, setIsGuideOpen ] = useState( false );
```

Comment:
Hey...

I payed attention to that requirement. 

That's why I split into BlockEditorTour and BlockEditorTourWrapper. I figured that the `BlockEditorTour` component could be used directly from the About menu without using this hook, as this hook is more related to the option persisted in the back-end. 

I made another refactor now and extracted the BlockEditorGuide component. I think we can just call the component directly and it will show up. Let me know if you find a problem with this approach.

Reply:
> I made another refactor now and extracted the BlockEditorGuide component. I think we can just call the component directly and it will show up. Let me know if you find a problem with this approach.

That seems good to me :) That seems lower effort and easier then adding a whole context.